### DISCUSSION TITLE
@nutui/nutui 打包测试说明

### DISCUSSION THREAD

**Discussion by richard1015** [jdf2e/nutui#1463] [Q&A]
1.  切换到[next](https://github.com/jdf2e/nutui)分支，安装依赖
2.  修改`package.json`  name字段修改一下 例如  `nutui-test-t1` (此包可能已被占用)
3. 执行 `npm run build` 代表构建 `@nutui/nutui` 类型的包，用于vue3 h5项目
4. 执行 `npm run build:taro:vue` 代表构建 `@nutui/nutui-taro` 类型的包，用于vue3+taro 多端小程序项目
5. 登录你自己的 npm 账号  npm login
6. 执行 npm run publish
7. 去[npmjs](https://www.npmjs.com/)官网搜索你发布的  `nutui-test-t1`

**Comment by zjyau** [comment]
THK!!!
